extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -74,6 +74,10 @@ declare module 'vscode-mssql' {
          */
         getDatabaseNameFromTreeNode(node: ITreeNodeInfo): string;
 
+		/**
```

Comment:
format

Reply:
not sure why this happened since the format looks aligned on my end. Will copy paste from mssql repo hope that fixes format.

Reply:
I think it's a tabs vs. spaces thing

Reply:
If you format the doc it should fix this kind of stuff automatically.

(but also turn on whitespace characters to easily see tabs vs spaces)